recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 
 build:
   number: 0
-  skip: True  # [py3k]
+  skip: True  # [osx and py3k]
```

Comment:
>  Is AppVeyor not expected to have a py3 build?

Since you had a global skip on Python 3 before you need to re-render to update the build matrix. Are you familiar with `conda-smithy` to do that?